### PR TITLE
MDCT-2914: FFY 2023 Content Changes (Section 3C)

### PR DESCRIPTION
**Pull request by karla-vm**
### Description

Content updates to reflect copy for FFY 2023 in Section 3C of CARTS.

### Related ticket(s)

MDCT-2914

---
### How to test

- Log in as `cms.admin@test.com`
- Generate form base templates for both 2022 and 2023
- Log in as `stateuser2@test.com`
- Navigate to the 2023 form, Section 3C

Verify that the content in red (on the ticket) matches the content in the form

### Important updates

N/A

---
### Author checklist

- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://bit.ly/3zPrxuZ) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

**Review by BearHanded — DISMISSED** [review]

File: services/database/data/seed/seed-section-base-2023.json [hunk 1]
```diff
@@ -2823,9 +2823,9 @@
                 "questions": [
                   {
                     "id": "2023-03-c-02-01",
-                    "hint": "Don’t include applicants being considered for redetermination — these data will be collected in Part 3.",
+                    "hint": "Don’t include applicants being considered for redetermination — these data will be collected in Part 3. \n\nPlease note: numbers reported in questions 2-4 of this part are a subset of the total reported in question 1. Therefore, the totals reported in questions 2-4 should be smaller than the number reported in question 1.",
```
Comment (gmrabian) [inline services/database/data/seed/seed-section-base-2023.json:2826]: First sentence here differs:
ticket – Don’t include applicants who are being considered for redetermination — these data will be collected in Part 3.
text – Don’t include applicants being considered for redetermination — these data will be collected in Part 3. 

diff: "who are"

File: services/database/data/seed/seed-section-base-2023.json [hunk 2]
```diff
@@ -3476,7 +3477,7 @@
...
+                "text": "Tracking a cohort of children enrolled in CHIP (Title XXI) will indicate how long a specific group of children stays enrolled over an 18-month period. This information is required by Section 402(a) of CHIPRA. \nTo track your cohort, identify a group of children ages 0 to 16 years who are newly enrolled in CHIP and/or Medicaid as of January through March 2022 (the second quarter of FFY 2022). Children in this cohort must be 16 years and 0 months or younger when they enroll to ensure they don't age out of the program by the end of the 18-month tracking period. \nYou’ll identify a new cohort every two years. Last year, you reported on the number of children at the start of the cohort (Jan — Mar 2022) and six months later (July — Sept 2022). This year, you'll report on the same cohort for their enrollment status 12 months later (Jan — Mar 2023) and 18 months later (July — Sept 2023). If data are unknown or unavailable, leave it blank — don’t enter a zero unless these data are known to be zero.",
```
Comment (gmrabian) [inline services/database/data/seed/seed-section-base-2023.json:3480]: "you'll report on the same cohort for their enrollment status 12 months later"

In the ticket "same cohort" here is italicized

File: services/database/data/seed/seed-section-base-2023.json [hunk 3]
```diff
@@ -5141,7 +5142,7 @@
...
+                "text": "Tracking a cohort of children enrolled in Medicaid (Title XIX) will indicate how long a specific group of children stays enrolled over an 18-month period. This information is required by Section 402(a) of CHIPRA. \nTo track your cohort, identify a group of children ages 0 to 16 years, who are newly enrolled in Medicaid and/or CHIP as of January through March 2022 (the second quarter of FFY 2022). Children in this cohort must be 16 years and 0 months or younger when they enroll to ensure they don’t age out of the program by the end of the 18-month tracking period. \nYou’ll identify a new cohort every two years. Last year, you reported on the number of children at the start of the cohort (Jan – Mar 2022) and six months later (July – Sept 2022). This year, you'll report on the same cohort for their enrollment status 12 months later (Jan – Mar 2023) and 18 months later (July – Sept 2023). If data is unknown or unavailable, leave it blank — don’t enter a zero unless the data is known to be zero.",
```
Comment (gmrabian) [inline services/database/data/seed/seed-section-base-2023.json:5145]: same note about italicizing "same cohort"

**Review by gmrabian — CHANGES_REQUESTED** [review]
Also want to call out the first paragraph of text in "Helpful hints on age groups" differs between the ticket and what's currently in CARTS (even though the ticket indicates no change)

**Comment by karla-vm** [comment]
> Also want to call out the first paragraph of text in "Helpful hints on age groups" differs between the ticket and what's currently in CARTS (even though the ticket indicates no change)

Great catch, I've updated this to match the ticket 🙏🏽

File: services/database/data/seed/seed-section-base-2023.json (hunk 3 above)
Comment (karla-vm) [reply services/database/data/seed/seed-section-base-2023.json:5145]: DK mentioned we won't need the italics in these hints, apologies for not calling that out before!

File: services/database/data/seed/seed-section-base-2023.json (hunk 2 above)
Comment (karla-vm) [reply services/database/data/seed/seed-section-base-2023.json:3480]: Same here.

**Review by gmrabian — DISMISSED** [review]

**Review by BearHanded — DISMISSED** [review]

**Review by BearHanded — APPROVED** [review]

**Pull request merged**